doc/classes/PathFollow3D.xml

```diff
@@ -16,7 +16,7 @@
 			There are two answers to this problem: either increase the number of cached points and increase memory consumption, or make a cubic interpolation between two points at the cost of (slightly) slower calculations.
 		</member>
 		<member name="h_offset" type="float" setter="set_h_offset" getter="get_h_offset" default="0.0">
-			The node's offset along the curve.
+			The node's offset perpendicular to the curve, to the right direction.
```

Comment:
I checked that and the old description is correct. The `h_offset` offsets the follower along the "forward" direction.